app/src/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require('https')
+const { parse } = require('url')
+const next = require('next')
+const fs = require('fs')
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handle = app.getRequestHandler()
+// @ts-ignore
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0
```

Comment:
Isn't the point of https so we can now remove this line?

Reply:
since the ddf cert is self signed, we still need this to get past that auth rejection

Reply:
we should turn this back on in production. If you use self signed certs graphql considers it unauthorized though: https://github.com/apollographql/apollo-link/issues/229